R/geom-text.r

```diff
@@ -80,18 +87,26 @@
 #'
 #' p + geom_text(aes(size = wt))
 #' # Scale height of text, rather than sqrt(height)
-#' p + geom_text(aes(size = wt)) + scale_radius(range = c(3,6))
+#' p +
+#'   geom_text(aes(size = wt)) +
+#'   scale_radius(range = c(3,6))
 #'
 #' # You can display expressions by setting parse = TRUE.  The
 #' # details of the display are described in ?plotmath, but note that
 #' # geom_text uses strings, not expressions.
-#' p + geom_text(aes(label = paste(wt, "^(", cyl, ")", sep = "")),
-#'   parse = TRUE)
+#' p +
+#'   geom_text(
+#'     aes(label = paste(wt, "^(", cyl, ")", sep = "")),
+#'     parse = TRUE
+#'   )
 #'
 #' # Add a text annotation
 #' p +
 #'   geom_text() +
-#'   annotate("text", label = "plot mpg vs. wt", x = 2, y = 15, size = 8, colour = "red")
+#'   annotate(
+#'     "text", label = "plot mpg vs. wt",
```

Comment:
This `"text",` can be placed in the same line of `annotate(`, but this is also fine.

> Short unnamed arguments can also go on the same line as the function name, even if the whole function call spans multiple lines.
https://style.tidyverse.org/syntax.html#long-lines

Reply:
I've noticed that Ctrl + Enter sometimes errors out at the bottom if the pattern is mismatched about code touching the parentheses or not

```r
# fine
annotate("text", 
         label = "plot mpg vs. wt")

# fine
annotate(
  "text",
  label = "plot mpg vs. wt"
)

# not fine
annotate("text",
  label = "plot mpg vs. wt"
)
```

Here's an example
```r
library(tidyverse)

mpg %>% 
  mutate(
    x1 = "ok here") %>% 
  mutate(
    x2 = "not ok") # use Ctrl + Enter in this line
````

![image](https://user-images.githubusercontent.com/14003618/86200900-f795d100-bb2b-11ea-85c6-e636908ebbad.png)

Reply:
Hmm, thanks for the information. I hope this will be fixed on RStudio...